engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterRunLoop.swift

```diff
@@ -103,8 +103,8 @@ import Foundation
   }
 
   // Schedules a block to be executed on the main thread.
-  @objc(performBlock:afterDelay:)
-  public func perform(block: @escaping () -> Void, afterDelay delay: TimeInterval) {
+  @objc(performWithDelay:block:)
+  public func perform(withDelay delay: TimeInterval, _ block: @escaping () -> Void) {
```

Comment:
I think we want to keep `block` explicit param. Similar API: https://developer.apple.com/documentation/foundation/timer/init(timeinterval:repeats:block:)

he other one `perform(_ block)` is probably fine since it's after a verb `perform`

Reply:
Done.